src/dawn/engine_dawn.cc

```diff
@@ -17,6 +17,7 @@
 #include <algorithm>
 #include <cassert>
 #include <cstdint>
+#include <string>
```

Comment:
Is this needed?

Reply:
No. I removed not needed `include` statements.